### ISSUE TITLE
Change the collision handling

### ISSUE THREAD

**Issue by LeoTrain** [LeoTrain/MarioIsaac#13]
For the moment it is working but not perfectly. We should try to find a better way to handle the collisions between the walls.

**Comment by hanneswirnsberger** [comment]
Yeah sometimes it is better to write things new from the scratch!

**Comment by LeoTrain** [comment]
https://www.youtube.com/watch?v=8PtKtyK-iVg&ab_channel=DaFluffyPotato

**Comment by hanneswirnsberger** [comment]
> I think it's better now, if you could run some tests to check i would really appreciate it !

Hehyo, i'll check today when i am back home :) 👍
